package.json

```diff
@@ -72,6 +72,8 @@
     "start": "EXTEND_ESLINT=true react-scripts start",
     "build": "EXTEND_ESLINT=true react-scripts build",
     "test": "react-scripts test",
+    "deploy:prod": "yarn build && firebase deploy --only hosting:prod-target",
```

Comment:
`scripts` can include a `pre` task that runs before the actual deploy by default, but separates the actual deploy (uploading to Firebase) and the necessary precondition (building).

```
"predeploy:prod": "yarn build",
"predeploy:staging": "yarn build",
"deploy:prod", ...
```

(See the package.json on Ovio Explore)

Also, does the static build involve "baking" in a URL for assets? Do we want another parameter for `yarn build` to specify a staging build vs. a production build?

Reply:
For now, there is no difference between a staging build and a production build ...